content/en/releases/patch-releases.md

```diff
@@ -92,6 +92,7 @@ End of Life for **1.23** is **2023-02-28**.
 
 | Patch Release | Cherry Pick Deadline | Target Date | Note |
 |---------------|----------------------|-------------|------|
+| 1.23.2        | 2022-01-14           | 2022-01-19  | [Out-of-Band Release](https://groups.google.com/u/2/a/kubernetes.io/g/dev/c/Xl1sm-CItaY) |
 | 1.23.2        | 2022-01-14           | 2022-01-19  |      |
```

Comment:
Maybe..?

```suggestion
| Patch Release | Cherry Pick Deadline | Target Date | Note |
|---------------|----------------------|-------------|------|
| 1.23.3        | 2022-01-24           | 2022-01-25  | [Out-of-Band Release](https://groups.google.com/u/2/a/kubernetes.io/g/dev/c/Xl1sm-CItaY) |
| 1.23.2        | 2022-01-14           | 2022-01-19  |      |
```

Reply:
I think I forgot to save the file and left the line just as it was copy + pasted, thanks @jihoon-seo !